pkgs/applications/science/electronics/verilog/default.nix

```diff
@@ -1,14 +1,14 @@
 { stdenv, fetchFromGitHub, autoconf, gperf, flex, bison }:
 
 stdenv.mkDerivation rec {
-  name = "iverilog-${version}";
-  version = "2019.03.27";
+  pname = "iverilog";
+  version = "2019.08.01";
```

Comment:
Can you please rename this to ``unstable-2019-08-01`` as per https://nixos.org/nixpkgs/manual/#sec-package-naming please?

Reply:
Oh yeah, thanks!